File: App-Revistas/src/app/Objetos/Reaccion.ts
export class Reaccion {

    idReaccion: number;
    reaccion: boolean;
    fecha: string;
    nombreUsuario: string;
    idRevista: number;

    constructor(
        idR: number,
        reac: boolean,
        fech: string,
        nombreU: string,
        idRev: number
    ){
        this.idReaccion = idR;
        this.reaccion = reac;
        this.fecha = fech;
        this.nombreUsuario = nombreU;
        this.idRevista = idRev;
    }
}

File: BackendRevistas/src/main/java/EntidadesPrincipales/Reaccion.java
package EntidadesPrincipales;

/**
 *
 * @author marco
 */
public class Reaccion {

    private int idReaccion;
    private boolean reaccion;
    private String fecha;
    private String nombreUsuario;
    private int idRevista;

    public Reaccion(int idReaccion, boolean reaccion, String fecha, String nombreUsuario, int idRevista) {
        this.idReaccion = idReaccion;
        this.reaccion = reaccion;
        this.fecha = fecha;
        this.nombreUsuario = nombreUsuario;
        this.idRevista = idRevista;
    }

    public int getIdReaccion() {
        return idReaccion;
    }

    public void setIdReaccion(int idReaccion) {
        this.idReaccion = idReaccion;
    }

    public boolean isReaccion() {
        return reaccion;
    }

    public void setReaccion(boolean reaccion) {
        this.reaccion = reaccion;
    }

    public String getFecha() {
        return fecha;
    }

    public void setFecha(String fecha) {
        this.fecha = fecha;
    }

    public String getNombreUsuario() {
        return nombreUsuario;
    }

    public void setNombreUsuario(String nombreUsuario) {
        this.nombreUsuario = nombreUsuario;
    }

    public int getIdRevista() {
        return idRevista;
    }

    public void setIdRevista(int idRevista) {
        this.idRevista = idRevista;
    }

}
